go-server-server/go/persistent.go

```diff
@@ -80,7 +80,7 @@ func InitialiseVariables() {
         now := time.Now().In(loc)
         ServerResetTime = fmt.Sprintf("%v", now)
 
-        newuuid, _ := uuid.NewV4()
+        newuuid := uuid.NewV4()
```

Comment:
Have to introduce this change because the uuid library behaves differently when consumed as a module https://github.com/golang/go/issues/34280#issuecomment-531116702